packages/acs-ui-common/review/acs-ui-common.api.md

```diff
@@ -7,7 +7,13 @@
 import { CommunicationIdentifier } from '@azure/communication-common';
 
 // @public (undocumented)
-export type AreEqual<A, B> = A extends B ? (B extends A ? true : false) : false;
+export type AreEqual<A extends (props: any) => JSX.Element | undefined, B extends (props: any) => JSX.Element | undefined> = true extends AreTypeEqual<A, B> & AreParamEqual<A, B> ? true : false;
```

Comment:
Could the `extends (props: any) => JSX.Element | undefined` be something like `extends React.Component`?

Reply:
Good catch! React.Component contains more than just functional component, which makes this expression a little bit tricky